zingo-testutils/src/chain_generics/fixtures.rs

```diff
@@ -419,16 +420,32 @@ where
         &mut environment,
         &primary,
         &secondary,
-        vec![(Shielded(shpool), 100_000)],
+        vec![(Shielded(shpool), 100_000 + make_change)],
     )
     .await;
 
     let tertiary = environment.create_client().await;
-    with_assertions::propose_send_bump_sync_recipient(
-        &mut environment,
-        &secondary,
-        &tertiary,
-        vec![(pool, 25_000)],
-    )
-    .await;
+    let expected_fee = if make_change == 0 {
+        fee_tables::one_to_one(shpool, pool, false)
+    } else {
+        fee_tables::one_to_one(shpool, pool, true)
+    };
+    // assert_eq!(
+    //     secondary
+    //         .propose_send_all(tertiary,
+    //         get_base_address(tertiary, pool))
+    //         .await
+    //         .into_u64(),
+    //     0
+    // );
+    assert_eq!(
+        expected_fee,
+        with_assertions::propose_send_bump_sync_recipient(
```

Comment:
Do we have any tests that exercise the `propose_send_all` logic?

Reply:
not yet